cost-analyzer/values.yaml

```diff
@@ -614,9 +612,12 @@ reporting:
   logCollection: true
   # Basic frontend analytics
   productAnalytics: true
+
   # Report Javascript errors
   errorReporting: true
   valuesReporting: true
+  # googleAnalyticsTag allows you to embed your Google Global Site Tag to track usage of Kubecost.
```

Comment:
I think this should be a paid feature since it's mostly an enterprise use case. Want to mark as such? Happy to discuss.

Reply:
Happy to do that - Do we need to do anything other than mark it here and implement the gate on the front end?

Reply:
Can discuss, but my vote is just mark for now!

Reply:
Doneski.